docs/apps/jax.md

```diff
@@ -8,42 +8,34 @@ tags:
 JAX is Autograd and XLA, brought together for high-performance machine
 learning research.
 
-!!! News
-
-    **5.10.2022** Due to Puhti's update to Red Hat Enterprise Linux 8
-    (RHEL8), **the number of supported JAX versions has been reduced.**
-    Please [contact our servicedesk](../support/contact.md) if you really
-    need access to older versions.
-
 
 ## Available
 
 Currently supported JAX versions:
 
-| Version | Module       | Puhti | Mahti | LUMI | Notes           |
-|:-------:|--------------|:-----:|:-----:|:----:|-----------------|
-| 0.4.20  | `jax/0.4.20` | X     | X     | -    | default version |
-| 0.4.18  | `jax/0.4.18` | -     | -     | X*   |                 |
-| 0.4.14  | `jax/0.4.14` | X     | X     | -    |                 |
-| 0.4.13  | `jax/0.4.13` | X     | X     | -    |                 |
-| 0.4.1   | `jax/0.4.1`  | X     | X     | -    |                 |
-| 0.3.13  | `jax/0.3.13` | X     | X     | -    |                 |
+| Version | Module             | Puhti | Mahti | LUMI | Notes           |
+|:-------:|--------------------|:-----:|:-----:|:----:|-----------------|
+| 0.4.23  | `jax/0.4.23-py3.9` | X     | X     | X*   | default version |
+| 0.4.20  | `jax/0.4.20`       | X     | X     | X*   |                 |
+| 0.4.18  | `jax/0.4.18`       | -     | -     | X*   |                 |
+| 0.4.14  | `jax/0.4.14`       | X     | X     | -    |                 |
+| 0.4.13  | `jax/0.4.13`       | X     | X     | -    |                 |
+| 0.4.1   | `jax/0.4.1`        | X     | X     | -    |                 |
+| 0.3.13  | `jax/0.3.13`       | X     | X     | -    |                 |
 
-The modules contain [JAX](https://github.com/google/jax/) for Python
-with GPU support via CUDA/ROCm.
+The modules contain [JAX](https://github.com/google/jax/) for Python 9
```

Comment:
I guess it should say Python 3.9 here, not "Python 9"

Reply:
good catch, fixed